src/Stratis.Bitcoin.Features.RPC/RPCFeature.cs

```diff
@@ -101,7 +101,7 @@ public override void Initialize()
             }
             else
             {
-                this.logger.LogWarning("RPC Server is off based on configuration.");
+                this.logger.LogDebug("RPC Server is off based on configuration.");
```

Comment:
I think this can maybe be useful to the user see this? So  perhaps `LogInformation` to the console?

Reply:
Fixed